docker-compose.yml

```diff
@@ -57,6 +57,13 @@ services:
     depends_on:
       - varnish
 
+  memcached:
+    build:
+      context: ./memcached/1.4/
+    image: quay.io/continuouspipe/memcached1.4:v1.0
```

Comment:
I think I prefer more quay.io/continuouspipe/memcached:1.4-1.0 (which is more like how rpms and debs are done)